backend/canvas_app_explorer/lti1p3.py

```diff
@@ -158,10 +158,17 @@ def create_user_in_django(request: HttpRequest, message_launch: ExtendedDjangoMe
     user_obj.backend = 'django.contrib.auth.backends.ModelBackend'
     django_login(request, user_obj)
 
-    if course_id is not None and isinstance(course_id, int):
-        request.session['course_id'] = course_id
+    course_id_error = Exception('The canvas_course_id custom LTI variable must be configured.')
+    if course_id is not None:
+        try:
+            course_id_int = int(course_id)
+        except ValueError:
+            logger.error(f'Course ID from LTI launch cannot be converted to an integer: {course_id}')
+            raise course_id_error
```

Comment:
I feel like the error msg should be different than the course_id_error. We can use the same message put in the log: 'Course ID from LTI launch cannot be converted to an integer: {course_id}'

Reply:
Okay, see https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/285/commits/da76ae33b3a6e57a95cf4a4addc15f53689854ce.